### DISCUSSION TITLE
Upgrade to beta verison

### DISCUSSION THREAD

**Discussion by jt7005** [awawa-dev/HyperHDR#262] [General]
Hi,
  I need help please, I trying to upgrade to the latest beta version of HYPERHDR but keep running into trouble.
Everything appears to go fine however I can get no web config page
I am using a Raspberry pi 3

The process I am doing is :
Download and extract the SD image from HyperHDR github release repository.  SD-card-image-rpi34-armv7.zip

Unzip and burn img to SD card
Fix up network settings etc
Then place the SD card in Pi, boot up
Do the normal app updates 
Everything works perfect at this point! Lights /webpage all great

**Now I go to upgrade**

I then remove hyperhdr
$sudo apt remove hyperhdr

All appears good so far

I then copy over HyperHDR-18.0.0.0beta1-Linux-armv7l.deb

sudo apt install ./HyperHDR-18.0.0.0beta1-Linux-armv7l.deb

all appear to be fine did not see any errors

Reboot

Pi boots fine

Pi is on the network 

I then 
sudo systemctl stop hyperhdr@pi.service
sudo systemctl disable hyperhdr@pi
sudo systemctl enable hyperhdr@root
Reboot

But I cannot get to webpage to config

Please where I am going wrong?

**Comment by awawa-dev** [comment]
Hi
You have done everything correctly. Unfortunately when v17 SD images were released, the current version was Debian Buster and now it's Bullseye. V18 beta versions are prepared for release using newer target. Probably if you run ```/usr/bin/hyperhdr``` it will complain about missing glibc 2.29 and  ```ldd -version``` will tell that you have glibc 2.28 despite system fully upgraded (as for Buster). And it's impossible for us to ship that core system library along side HyperHDR: it won't work anyway on older systems.  
  
Solution:  
1) Finish full migration from Buster to Bullseye https://www.tomshardware.com/how-to/upgrade-raspberry-pi-os-to-bullseye-from-buster (modifying /etc/apt/sources.list and later apt update/upgrade again is important as otherwise it won't find newer glibc in older repo). Tested and it worked for me.    
2) Alternatively download official new Raspberry Pi OS lite from https://www.raspberrypi.com/software/operating-systems/#raspberry-pi-os-32-bit and make new installation. Beware that they have recently removed default pi user/password and headless installation is a lot more complicated if you are not using their installer.

**Reply by jt7005** [reply]
Thank you very much, you were spot on. I now have a working ver 18 Hyperhdr
Thanks again
